### ISSUE TITLE
LineLengthCheck - line.length() differs from columnNo.get()

### ISSUE THREAD

**Issue by sebastiaanspeck** [StevenLooman/sonar-magik#33]
```
_method query_dialog.datasets
	soc              << _self.spatial_object_controller
	allowed_datasets << rope.new_from(soc.browsable_datasets).select(predicate.using(_proc(x) >> x.collection(:swif!info_flag) _isnt _unset _endproc))

	>> allowed_datasets
_endmethod
$
```

Results in `Line is too long (147/120) (line-length)`

```
_method query_dialog.build_gui(a_container)
	description_text_container << sw_container.new(a_container, 1, 2, :col_resize_values, {25, 75}, :row_alignment, :top)
_endmethod
$
```
Results in `Line is too long (118/120) (line-length)`

It is correct that the line is longer than 120, if I look at the first code block, the column number is 154. 
Looking at the check, we do use the column number to check if the line is too long:
https://github.com/StevenLooman/magik-tools/blob/dc35a9ca5f18530575b5f7e32239c2ccf5c98ade/magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/LineLengthCheck.java#L63

But for creating the `endPosition` and the `message` we use `line.length()`. Why not use `columnNo.get()`?

**Comment by StevenLooman** [comment]
There is a difference between what the editor "sees" and what a user sees. The editor sees a tab as a single character, while a user sees a tab as (up to) 8 characters (when tab width is configured as 8, which is often used in Smallworld code).

The `columnNo` should be used for the message, not the `line.length()`.

**Comment by sebastiaanspeck** [comment]
Fixed by af627a8f2ddc13f3739b69d474b05dc22339df0a
